asfyaml/feature/github/branch_protection.py

```diff
@@ -285,10 +283,11 @@ def branch_protection(self: ASFGitHubFeature):
             protection_changes[branch] = branch_changes
 
     # remove branch protection from all remaining protected branches
-    for branch_name, branch in protected_branches.items():
```

Comment:
the branch variable is used in the next line, so shouldn't be omitted

Reply:
hmm I missed that, it still worked, though it showed wrong information.
Fixed, ty.